packages/ripple-binary-codec/.eslintrc.js

```diff
@@ -103,19 +105,19 @@ module.exports = {
     'max-lines-per-function': 'off',
     'require-unicode-regexp': 'off',
     'no-undef-init': 'off',
-    'curly': 'off',
```

Comment:
Why do only some of these need quotes?

Reply:
I don't understand the question

Reply:
I'm not sure why, but I think that these are changes that `prettier` makes when you save.

Reply:
Looks like the ones with the dashes need quotes and the ones without don't